scripts/tunnelstat

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python3
+
+#####################################################################
+#
+# tunnelstat is a tool for summarizing various tunnel statistics.
+#
+#####################################################################
+
+import _pickle as pickle
+import argparse
+import datetime
+import sys
+import os
+import sys
+import time
+
+# mock the redis for unit test purposes #
+try:
+    if os.environ["UTILITIES_UNIT_TESTING"] == "2":
+        modules_path = os.path.join(os.path.dirname(__file__), "..")
+        test_path = os.path.join(modules_path, "tests")
+        sys.path.insert(0, modules_path)
+        sys.path.insert(0, test_path)
+        import mock_tables.dbconnector
+except KeyError:
+    pass
+
+from collections import namedtuple, OrderedDict
+from natsort import natsorted
+from tabulate import tabulate
+from utilities_common.netstat import ns_diff, table_as_json, STATUS_NA, format_brate, format_prate
+from swsscommon.swsscommon import SonicV2Connector
+
+
+nstat_fields = ("rx_b_ok", "rx_p_ok", "tx_b_ok", "tx_p_ok")
+NStats = namedtuple("NStats", nstat_fields)
+
+header = ['IFACE', 'RX_OK', 'RX_BYTES', 'RX_PPS','TX_OK', 'TX_BYTES', 'TX_PPS']
```

Comment:
would it be better to change TX_OK to TX_PKT as we are using bytes for the other column ? Same for RX_OK and RX_PKT